aws/logs_monitoring/forwarder.py

```diff
@@ -25,92 +28,160 @@
     DD_PORT,
     DD_TRACE_INTAKE_URL,
     DD_FORWARD_LOG,
+    DD_RETRY_EVENTS,
     SCRUBBING_RULE_CONFIGS,
     INCLUDE_AT_MATCH,
     EXCLUDE_AT_MATCH,
+    RETRY_INTERVAL_SECONDS,
```

Comment:
should this env variable be documented in the README ?

Reply:
👍 I'll added also to the template